Kickstarter-iOS/Features/RewardsCollection/Controller/RewardsCollectionViewController.swift

```diff
@@ -100,6 +100,7 @@ final class RewardsCollectionViewController: UICollectionViewController {
     self.setupConstraints()
 
     self.viewModel.inputs.viewDidLoad()
+    self.viewModel.inputs.shippingRuleSelected(nil)
```

Comment:
This is a workaround that I'm not sold on. The selected ShippingRule needs to emit a signal for the view model outputs that trigger navigation to the add-ons or pledge screens to complete. However, in the case where there are only rewards without shipping (Digital or Local pickup only), the Shipping Dropdown will not appear on the screen. Therefore, a ShippingRule will not be selected, and the navigation outputs will not complete.

So the solution here is to emit `nil` on viewDidLoad so they can. I imagine there is a way to handle this in ReactiveSwift that I'm missing, but for now, I'm moving forward with this approach to keep the development going.